kubeflow/fairing/builders/cluster/minio_context.py

```diff
@@ -4,7 +4,7 @@
 from kubeflow.fairing import utils
 from kubeflow.fairing import constants
 
-constants.constants.KANIKO_IMAGE = "gcr.io/kaniko-project/executor:v0.14.0"
+constants.constants.KANIKO_IMAGE = "gcr.io/kaniko-project/executor:v0.22.0"
```

Comment:
@iancoffey Could you please enhance here, do not define here (remove the `constants.constants.KANIKO_IMAGE`), directly import the value that's definied in `constants`? Thanks.

Reply:
removed that dupe, thanks